notifications/models/govuk-notify-model.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable consistent-return */
+const config = require('../../config');
+const logger = require('../../lib/logger')({ env: config.env });
+const {
+  caseworkerEmail,
+  successTemplateId,
+  failureTemplateId
+} = config.notifications;
+
+module.exports = class NotifyModel {
+  async sendCaseworkerNotification(client, jobReport) {
+    const {
+      success,
+      recordsCount,
+      errorMessage,
+      fileUploadTime,
+      jobEndedTime,
+      invalidRecords
+    } = jobReport;
+
+    const emailProps = {
+      csv_uploaded_datetime: this.getDateAndTimeString(fileUploadTime),
+      cepr_update_datetime: this.getDateAndTimeString(jobEndedTime),
+      has_invalid_records: (invalidRecords.length ? 'yes' : 'no'),
+      link_to_file: ''
+    };
+    let templateId;
+    if (success) {
+      emailProps.records_count = recordsCount;
+      templateId = successTemplateId;
+    } else {
+      emailProps.failure_message = errorMessage;
+      templateId = failureTemplateId;
+    }
+
+    if (invalidRecords.length) {
+      const invalidRecordsCsv = Buffer.from(this.writeInvalidRecordsToCsv(invalidRecords), 'utf8');
+      const fileNameDate = this.getDateAndTimeString(fileUploadTime).replace(/[\/\s:]/g, '-');
```

Comment:
no biggie, just an alternative more clearer to me

```
const moment = require('moment');

// Assuming fileUploadTime is a string representing a date and time
const formattedDateTime = moment(fileUploadTime).format('YYYY-MM-DD-HH-mm-ss');

console.log(formattedDateTime);
```

Reply:
Yes I thought of this, but [moment is probably at end of life](https://momentjs.com/docs/#/-project-status/) so I didn't want to include it in a new project.